packages/vitaminjs-jest/src/babelrcJestTransformer.js

```diff
@@ -0,0 +1,13 @@
+import { createTransformer } from 'babel-jest';
+
+const babelConfig = {
+    presets: [
+        ['env', {
+            targets: { node: 'current' },
```

Comment:
Having `current` here makes produced build code not portable. Node version compatibility is not deterministic depending on who built this. By the way, I'm not sure we should check-in build artifacts into source control.

Reply:
should be based on the user package's engines config :)